docs/detections/rules-ui-create.asciidoc

```diff
@@ -355,6 +355,11 @@ also affect this rule.
 .. *Building block* (optional): Select to create a building-block rule. By
 default, alerts generated from a building-block rule are not displayed in the
 UI. See <<building-block-rule>> for more information.
+.. *Indicator prefix override*: Specify the default threat indicator path (`threat_indicator_path`) that the indicator match rule will use when obtaining threat indicator data from the default threat intelligence index. This data is used to enrich indicator match alerts with metadata about matched threat indicators. The default value for this field is `threat.indicator`. If your threat indicator data is at a different location, update this setting to ensure the correct field prefix is used so alert enrichment can still be performed.
```

Comment:
first sentence - indicator match rule will use threat_indicator_path to obtain indicator data NOT from the default threat index, but from the indicators obtained from the value of `indicator index` specified in the rule, could be default or custom.

Reply:
Ah yes, the IM rule queries threat indicator indices defined on the **Indicator index patterns** field